tools/toitlsp/lsp/toitdoc/toitdoc.go

```diff
@@ -184,10 +187,17 @@ func (b *builder) build() *Doc {
 		Libraries:  Libraries{},
 	}
 
+	packageURI := uri.PathToURI(b.rootPath) + "/.packages/"
 	for u, m := range b.summaries {
 		if b.excludeSDK && strings.HasPrefix(string(u), string(b.sdkURI)) {
 			continue
 		}
+		// TODO(florian): this is a complete hack.
```

Comment:
Maybe elaborate a tiny bit on what makes it a hack?

Reply:
Initially it was `Contains(stringu(u), "/.packages/")`. I then improved on it, but forgot to remove the TODO.